IBAnimatable-Material/AnimatableMaterialCardViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AnimatableMaterialCardViewController.swift
+//  IBAnimatable-Material
+//
+//  Created by George Kye on 2016-07-13.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import IBAnimatable
+import Material
+
+class AnimatableMaterialCardViewController: UIViewController {
```

Comment:
Since it is a demo ViewController, shall we can it `AnimatableMaterialCardViewController`. It looks like an element for the library. But may not be a problem if we have separate targets for library and demo app.